cibuildwheel/windows.py

```diff
@@ -32,32 +44,49 @@ def get_python_path(config):
         )
 
 
+def get_nuget_args(configuration):
+    if configuration.exact_version is None:
+        return None
+    python_name = "python" if configuration.version[0] == '3' else "python2"
+    if configuration.arch == "32":
+        python_name = python_name + "x86"
+    return [python_name, "-Version", configuration.exact_version, "-OutputDirectory", "C:/python"]
+
 def get_python_configurations(build_selector):
-    PythonConfiguration = namedtuple('PythonConfiguration', ['version', 'arch', 'identifier', 'path'])
+    PythonConfiguration = namedtuple('PythonConfiguration', ['version', 'arch', 'identifier', 'path', "exact_version"])
```

Comment:
Perhaps rename `exact_version` to `nuget_version`, since I don't think it's accurate in e.g. Appveyor

Reply:
Agree